app/controllers/collections_controller.rb

```diff
@@ -165,4 +165,22 @@ def destroy
     redirect_to(collections_url)
   end
 
+  private
+
+  def collection_params
+    params.require(:collection).permit(
+      :name, :title, :email, :header_image_url, :description,
```

Comment:
@zz9pzza I think we can take out `:icon_file_name`. I experimented, and if we allow it, the user can add an `icon_file_name` field to the form and set whatever name they want, but if we take it off this list of allowed parameters and they try to do that, it just ignores what the user has entered and takes the filename from the icon. I'm pretty sure that's exactly what's supposed to happen with strong parameters.